maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -141,7 +151,8 @@ def _init_pms(self):
             pm.set_init_state(
```

Comment:
The exploration of the specific SKU setting can be postponed.

You can think about the design of multiple SKU supporting first.